platform/win.lua

```diff
@@ -43,8 +43,9 @@ self.curl_request = function(url, request_json, completion_fn)
         '-X',
         'POST',
         '--data-binary',
-        table.concat { '@', curl_tmpfile_path }
+        --table.concat { '@', curl_tmpfile_path }
     }
+    table.insert(args, request_json)
```

Comment:
Looks like this change was made by mistake.

Reply:
Apologies, I don't quite remember the intention of this change either. I'll remove it and verify.

Reply:
I actually believe this was actually to prevent overwriting due to asynchronous calls? When updating multiple cards, only the last updateNoteFields goes through. I'm not 100% sure on this as I never got to the bottom of it, but I suspect it has something to do with using a temp txt file + mp.command_native_async. When I changed it to include the request_json at runtime (as the nix.lua also does), multiple card updates works again. Is there a reason for using a temporary text file for the curl commands?

Reply:
> Is there a reason for using a temporary text file for the curl commands?

Curl on windows doesn't work reliably when json is passed as a command line argument. Using a temporary file is a widely used solution. We also ran into the issue in the past and had to adopt the fix.

* https://curl.se/mail/archive-2018-05/0011.html

>  When updating multiple cards, only the last updateNoteFields goes through. 

Then we need to use a unique filename every time.